### DISCUSSION TITLE
Map on Virtual Cockpit

### DISCUSSION THREAD

**Discussion by giacomotri** [Mr-MIBonk/M.I.B._More-Incredible-Bash#220] [Q&A]
Hi, I activated the maps with mib 3.5.1 on q2 version MHI2_ER_AU276_P5515_MU1488.  On the Virtual cockpit the maps are not active, how can I activate them?

**Comment by harman-f** [comment]
Please share LOG and backup.

**Comment by Raikkonen85** [comment]
Something new About actívate nav in virtual cockpit? Audi a3 8Y mib3
